### ISSUE TITLE
508-defect-2 [SCREENREADER]: Feedback form - Text area requires user to set focus and click to start typing

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26567]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I should be able to click on a textarea label or set focus on the screen reader and start typing immediately.

## Details

The `<textarea>` requires me to take action beyond setting keyboard focus to enter data in MacOS VoiceOver + Safari and NVDA + Chrome/Firefox. This appears to be caused by either a DIV inside the textarea or custom JavaScript. Screenshot and proposed code snippet below.

## Acceptance Criteria

- [ ] Clicking the label sets focus on the text area and allows user to start typing immediately
- [ ] Users can press `TAB` to set focus on the text area and start typing immediately
- [ ] Ideally, move the `<p>` of helper text before the text area, so it's read after the label and a short pause

## Steps to Recreate

1. Open MacOS Safari + VoiceOver
2. Open the Feedback form
3. Press `CTRL + OPT + RIGHT_ARROW` until your virtual cursor is on the text area
4. Start typing
5. Verify no text is added to the text area

## Proposed Solution (if known)
Semantic HTML should clean this up and restore default browser and screen reader behavior.

```html
<div>
  <label for="task-entry-field">
    What task were you trying to do today?
  </label>
  <p id="hint-text">
    Please don't enter personal information into the text box. This includes your name, address,
    Social Security Number, medical information or anything else that someone could use to identify
    you.
  </p>
  <textarea
    aria-describedby="hint-text"
    id="task-entry-field"
    rows="3"
    validator-required="false"
  ></textarea>
</div>
```

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

<img width="429" alt="Screen Shot 2021-06-24 at 11 10 05 AM" src="https://user-images.githubusercontent.com/934879/123296988-c05ac580-d4dc-11eb-8e14-2c2da4dcacb9.png">
